ingestion_tools/scripts/common/mesh_converter.py

```diff
@@ -15,21 +17,91 @@ def convert_mesh_to_glb(convert_func: typing.Callable[[str], trimesh.Trimesh]) -
     :param convert_func: the function to convert a mesh file to a trimesh object.
     """
 
-    def wrapper(input_file: str, output_file: str, scale_factor: float = 1.0) -> None:
+    def wrapper(input_file: str, output_file: str, scale_factor: float = 1.0, *args, **kwargs) -> None:
         """
         Convert a mesh file to a glb file and scale it by a given factor.
 
         :param input_file: the path to a mesh file to convert.
         :param output_file: the name of the output file.
         :param scale_factor: the factor to scale the mesh by.
         """
-        mesh: trimesh.Trimesh = convert_func(input_file)
+        mesh: trimesh.Trimesh = convert_func(input_file, *args, **kwargs)
         mesh.apply_scale(scale_factor)
         mesh.export(output_file)
 
     return wrapper
 
 
+def check_mesh_name(input_file: str, name: str) -> str:
+    with h5py.File(input_file, "r") as fp:
+        for mesh_list in fp["segment_list/"].keys(): # noqa
+            # Mesh name
+            mesh_name_raw: bytes = fp[f"segment_list/{mesh_list}/biological_annotation/name"][()]
+            if not mesh_name_raw:
+                logger.warning("No mesh name found at segment_list/{mesh_list} in the hff file")
+            else:
+                mesh_name: str = mesh_name_raw.decode()
+                if mesh_name == name:
+                    return mesh_list
+        raise ValueError(f"Mesh name {name} not found in the hff file")
```

Comment:
There not being a mesh for a specific name, is a valid case. This should be caught by the `is_valid` check and not processed. So, we shouldn't throw and error here. But, it would be good to have a warning log.

Reply:
should is_valid return true or false if it is not found?

Reply:
Yes, that is correct.